src/main/resources/assets/embeddiumplus/lang/zh_cn.json

```
@@ -0,0 +1,51 @@
+{
+  "embeddium.plus.options.plus": "附加",
+  "embeddium.plus.options.common.off": "关",
+  "embeddium.plus.options.common.simple": "简单",
+  "embeddium.plus.options.common.advanced": "高级",
+  "embeddium.plus.options.common.slow": "慢",
+  "embeddium.plus.options.common.fast": "快",
+  "embeddium.plus.options.common.faster": "更快",
+  "embeddium.plus.options.common.realtime": "实时",
+  "embeddium.plus.options.displayfps.title": "显示FPS",
+  "embeddium.plus.options.displayfps.desc": "显示当前FPS. 高级模式还显示最低FPS以及15秒的平均FPS，这对判断性能很有用。",
+  "embeddium.plus.options.displayfps.position.title": "FPS显示位置",
+  "embeddium.plus.options.displayfps.position.desc": "将FPS显示偏移几个像素",
+  "embeddium.plus.options.darkness.title": "真正的黑暗",
+  "embeddium.plus.options.darkness.desc": "使世界其他地方更加逼真地黑暗。不影响日间照明或火把照明。",
+  "embeddium.plus.options.darkness.mode.title": "真实黑暗方式",
+  "embeddium.plus.options.darkness.mode.desc": "控制什么样的黑暗被视为真正的黑暗。",
+  "embeddium.plus.options.darkness.mode.pitchblack": "漆黑一片",
+  "embeddium.plus.options.darkness.mode.reallydark": "真实黑暗",
+  "embeddium.plus.options.darkness.mode.dark": "黑暗",
+  "embeddium.plus.options.darkness.mode.dim": "昏暗",
+  "embeddium.plus.options.chunkfadeinquality.title": "区块渐变质量",
+  "embeddium.plus.options.chunkfadeinquality.desc": "控制区块消退的速度。没有性能消耗，Fancy只需要更长的时间，但看起来有点酷。当前不工作",
+  "embeddium.plus.options.fog.title": "启用雾",
+  "embeddium.plus.options.fog.desc": "关闭整个世界中的所有雾",
+  "embeddium.plus.options.jei.title": "在搜索之前隐藏JEI",
+  "embeddium.plus.options.jei.desc": "关闭JEI项目，除非您搜索某些内容。按空格键可搜索所有内容。",

```

只是建议
Just a suggestion
```suggestion
  "embeddium.plus.options.jei.title": "不在搜索时隐藏JEI",
  "embeddium.plus.options.jei.desc": "隐藏JEI物品列表中的物品，直到你搜索了某些内容。输入空格可以搜索到所有内容。",

```
```suggestion
  "embeddium.plus.options.jei.title": "隐藏JEI直到开始搜索",
  "embeddium.plus.options.jei.desc": "使JEI不渲染物品列表中的物品，直到你搜索了某些内容。输入空格可以搜索到所有内容。",

```